spec/javascript/components/lease_ups/LeaseUpApplicationsPage.test.js

```diff
@@ -149,23 +150,23 @@ describe('LeaseUpApplicationsPage', () => {
       expect(mockCreateFieldUpdateComment.mock.calls.length).toEqual(0)
     })
 
-    test('Should call createFieldUpdateComment and close on successful submit', async () => {
+    test('should call createFieldUpdateComment and close on successful submit', async () => {
       mockCreateFieldUpdateComment.mockReturnValueOnce(true)
       openStatusModal(wrapper)
 
       const status = await wrapper.find(statusSelector).first().html()
       if (!status.toLowerCase().includes('processing') && !status.toLowerCase().includes('lease signed')) {
         expect(wrapper.find(subStatusSelector).exists()).toBe(true)
         const emptyStatus = wrapper.find(`${subStatusSelector} button`).html()
-        expect(emptyStatus.toLowerCase().includes('select one...')).toBe(true)
+        expect(emptyStatus.toLowerCase()).toContain('select one...')
 
         wrapper.find(subStatusSelector).find('button').simulate('click')
         await tick()
         wrapper.find(`${subStatusSelector} .dropdown-menu li a`).first().simulate('click')
         await tick()
 
         const updatedStatus = wrapper.find(`${subStatusSelector} button`).html()
-        expect(updatedStatus.toLowerCase().includes('select one...')).toBe(false)
+        expect(updatedStatus.toLowerCase()).not.toContain('select one...')
```

Comment:
these are going to have to be re-done on my PR since I moved the tests to another file. I can make the changes now

Reply:
actually I don't have these checks in there anymore so it should be fine